pkgs/development/python-modules/dtaidistance/default.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, llvmPackages
+, cython
+, numpy
+}:
+
+buildPythonPackage rec {
+  pname = "dtaidistance";
+  version = "2.3.10";
+
+  src = fetchFromGitHub {
+    owner = "wannesm";
+    repo = pname;
```

```suggestion
    repo = "dtaidistance";
```
It is not really semantically correct, we try to avoid `repo = pname;`